### PR TITLE
fix(vpc)!: gate NAT-gateway resources on IGW state

### LINKED ISSUE #384: bug(vpc): NAT-gateway routes and public route-table association crash plan when enable_internet_gateway=false or public_subnets_list is empty
## Affected module path
`modules/aws/vpc`

## OpenTofu / provider versions
- OpenTofu v1.12.4
- hashicorp/aws provider v6.54.0 (module requires `>= 6.0.0`)

## Summary
Several resources in `modules/aws/vpc/main.tf` compute their `count` from `var.enable_nat_gateway` and/or subnet-list length alone, without also checking `local.enable_igw` (which is `var.enable_internet_gateway && length(var.public_subnets_list) != 0`). Since NAT gateways themselves (`aws_nat_gateway.natgw`) are correctly gated on `local.enable_igw`, this mismatch causes a plan-time crash whenever a caller sets `enable_internet_gateway = false` (or supplies an empty `public_subnets_list`) while leaving `enable_nat_gateway` at its default `true`.

Affected resources (their `count` expressions do not account for `local.enable_igw`):
- `aws_route.private_default_route_natgw` (main.tf ~line 312-317)
- `aws_route.db_default_route_natgw` (main.tf ~line 333-338)
- `aws_route.dmz_default_route_natgw` (main.tf ~line 354-359)
- `aws_route.mgmt_default_route_natgw` (main.tf ~line 375-380)
- `aws_route.workspaces_default_route_natgw` (main.tf ~line 396-401)
- `aws_route_table_association.public` (main.tf ~line 418-422), which indexes `aws_route_table.public_route_table[0]` unconditionally even though that resource's own `count` is gated on `local.enable_igw`

## Repro steps
From `modules/aws/vpc`, run `tofu test` against a plan with:
```hcl
name                    = "core-vpc"
enable_internet_gateway = false
```
(all other variables left at their defaults, so `enable_nat_gateway = true` and `private_subnets_list`/`public_subnets_list` are non-empty).

A minimal reproducing test case is included (commented, not asserting a passing plan) in the PR that reported this issue: see `modules/aws/vpc/tests/vpc.tftest.hcl`, run `enable_internet_gateway_false_disables_igw_and_dependent_nat`.

## Expected vs actual behavior
- Expected: with `enable_internet_gateway = false`, the module should still produce a valid plan. Since NAT gateways require an IGW/public subnet to attach to, the NAT-gateway-dependent routes (and the public route-table association) should also be skipped (count 0) in this configuration, consistent with `aws_nat_gateway.natgw` itself being skipped.
- Actual: `tofu plan`/`tofu test` crashes with `Call to function "element" failed: cannot use element function with an empty list.` (for the `*_default_route_natgw` resources referencing `aws_nat_gateway.natgw[*].id`) and `The given key does not identify an element in this collection value: the collection has no elements.` (for `aws_route_table_association.public` referencing `aws_route_table.public_route_table[0]`).

## Actual tofu test error output (excerpt)
```
Error in function call

  on main.tf line 315, in resource "aws_route" "private_default_route_natgw":
 315:   nat_gateway_id         = element(aws_nat_gateway.natgw[*].id, count.index)
    +-------------------
    | while calling element(list, index)
    | aws_nat_gateway.natgw is empty tuple
    | count.index is 1

Call to function "element" failed: cannot use element function with an empty list.
```
```
Error: Invalid index

  on main.tf line 420, in resource "aws_route_table_association" "public":
 420:   route_table_id = aws_route_table.public_route_table[0].id

The given key does not identify an element in this collection value: the collection has no elements.
```

## Suggested acceptance criteria
- The `count` expressions for the 5 `*_default_route_natgw` resources incorporate the same "NAT gateway actually created" condition as `aws_nat_gateway.natgw` itself (i.e. also require `local.enable_igw`), so they are skipped when NAT gateways are skipped.
- `aws_route_table_association.public`'s `count` also incorporates `local.enable_igw`, so it is skipped when the public route table itself is skipped.
- A `tofu test` case with `enable_internet_gateway = false` (NAT gateway left at its default-enabled value) plans successfully with 0 NAT-gateway-dependent routes and 0 public route-table associations.
- A `tofu test` case with `public_subnets_list = []` (NAT gateway left enabled) plans successfully for the same reason.

## Reported by
Automated native-test-coverage effort (see PR adding `modules/aws/vpc/tests/`).

### PR DESCRIPTION
**Pull request by zachreborn**
# Description
Split out of the native-test-coverage PR (#392) per review feedback — that PR is meant to be test-only, so the functional fixes it surfaced belong here instead.

- `modules/aws/vpc/main.tf`: five `*_default_route_natgw` route resources and `aws_route_table_association.public` gated their `count` on `enable_nat_gateway`/subnet-list length alone, without also checking `local.enable_igw` the way `aws_nat_gateway.natgw` itself does. This crashed `plan` (`element` on an empty list / invalid index) whenever `enable_internet_gateway = false` (or `public_subnets_list = []`) was combined with the default `enable_nat_gateway = true`. Introduced `local.enable_natgw = var.enable_nat_gateway && local.enable_igw` and applied it consistently, including to `aws_eip.nateip` and `aws_nat_gateway.natgw` (the `*_default_route_natgw` gating itself was independently fixed upstream by #415 while this PR was in review; this branch was rebased onto that fix).
- `modules/aws/transit_gateway/connect/outputs.tf`: fixed a typo (`insider_cidr_blocks` → `inside_cidr_blocks`) in the `configurations` output. The misspelled key is kept as a deprecated backward-compatible alias so existing callers are not broken, and the deprecation is now documented in the public output description (not just an implementation comment).
- `modules/aws/flow_logs`: this module gains both new outputs and a functional, **breaking** behavior change:
  - Outputs: added `flow_log_ids`, `flow_log_eni_ids`, `flow_log_subnet_ids`, `flow_log_vpc_ids`, `flow_log_transit_gateway_ids`, and `flow_log_transit_gateway_attachment_ids` (additive, non-breaking) so callers can verify their flow-log target wiring for every supported target type. Also fixed the `arn` output, which returned the log group's `id` instead of its `arn`.
  - Behavior change: the module now enforces an explicit "exactly one of `flow_eni_ids`, `flow_subnet_ids`, `flow_transit_gateway_ids`, `flow_transit_gateway_attachment_ids`, `flow_vpc_ids`" contract via a `lifecycle.precondition` on `aws_kms_key.key`, rather than relying on an unguarded `coalesce()` call to error out.
  - **Breaking**: `lifecycle.precondition` requires Terraform >= 1.2, so `required_version` was bumped from `>= 1.0.0` to `>= 1.2.0`. Callers still on Terraform 1.0/1.1 (OpenTofu users are unaffected — OpenTofu forked after Terraform 1.6) will need to upgrade before consuming this module.
- Regenerated `terraform-docs` for `flow_logs` and `transit_gateway/connect`.

Not in scope for this PR: AWS provider `>= 6.25.0` (already this module's minimum) added a sixth valid `aws_flow_log` target, `regional_nat_gateway_id`. Adding support for it is a genuine new feature (variable, output, tests, docs), not a bug fix, so it's deferred to a separate `feat(flow_logs)` PR rather than expanding this one's scope further.

## Issue or Ticket
Fixes #384

## Type of change
- [x] `fix` — a bug fix (SemVer PATCH)

## Breaking Changes
- [x] Yes — this is a breaking change
- [ ] No

`modules/aws/flow_logs` `required_version` is now `>= 1.2.0` (previously `>= 1.0.0`), required for the new `lifecycle.precondition` enforcing the "exactly one target" contract. Terraform 1.0/1.1 callers of this module must upgrade Terraform before consuming this version. OpenTofu users are unaffected.

BREAKING CHANGE: modules/aws/flow_logs now requires Terraform >= 1.2.0 (previously >= 1.0.0) due to its new lifecycle.precondition-based "exactly one target" validation.

## TODOs
- [x] PR title follows Conventional Commits (`<type>(scope)!: description`).
- [x] Validate your code matches the style of the project (`tofu fmt -recursive`).
- [x] Update the docs (regenerated the `terraform-docs` block for `flow_logs` and `transit_gateway/connect`).
- [x] Validate all tests run successfully, including pre-commit checks.
- [x] Include release notes and description.

## Testing
Ran `tofu init -backend=false && tofu test` in the affected module directories:
- `modules/aws/vpc`: 38 passed, 0 failed
- `modules/aws/transit_gateway/connect`: 8 passed, 0 failed
- `modules/aws/flow_logs`: 7 passed, 0 failed (source-resolution/precondition behavior, including the exactly-one and no-target failure cases, and every target-type output)

`tofu fmt -check -diff -recursive` is clean.

Co-Authored-By: Oz <oz-agent@warp.dev>

### CODE REVIEW

File: modules/aws/transit_gateway/connect/README.md [hunk 1]
```diff
@@ -115,27 +115,27 @@ No modules.
...
 | <a name="input_transit_gateway_default_route_table_propagation"></a> [transit\_gateway\_default\_route\_table\_propagation](#input\_transit\_gateway\_default\_route\_table\_propagation) | (Optional) Boolean whether the Connect should propagate routes with the EC2 transit gateway propagation default route table. This cannot be configured or perform drift detection with Resource Access Manager shared EC2 Transit Gateways. Default value: true. | `bool` | `true` | no |
 | <a name="input_transit_gateway_id"></a> [transit\_gateway\_id](#input\_transit\_gateway\_id) | (Required) Identifier of EC2 transit gateway. | `string` | n/a | yes |
-| <a name="input_transport_attachment_id"></a> [transport\_attachment\_id](#input\_transport\_attachment\_id) | (Required) The underlying transit gateway VPC attachment ID. | `string` | n/a | yes |
+| <a name="input_transport_attachment_id"></a> [transport\_attachment\_id](#input\_transport\_attachment\_id) | (Required) The underlaying transit gateway VPC attachment ID. | `string` | n/a | yes |
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/transit_gateway/connect/README.md]: Fixed already in abac003c (fix(flow_logs): fix CI failures (codespell typo, tflint coalesce crash)) — variables.tf and the regenerated README now say 'underlying'.

File: modules/aws/flow_logs/main.tf [hunk 2]
```diff
@@ -19,13 +19,18 @@ data "aws_region" "current" {}
 ###########################
 locals {
 
-  flow_logs_source = coalesce(
+  # try() guards against the case where none of the five target variables are
+  # set (all null): coalesce() would otherwise fail with "no non-null,
+  # non-empty-string arguments". Real callers always provide exactly one
+  # target list; this only matters for static analysis tools (e.g. tflint)
+  # that evaluate the module with all variables left at their defaults.
+  flow_logs_source = try(coalesce(
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/main.tf:32]: Fixed in 50a41bf9: added a lifecycle precondition on aws_kms_key.key so a real plan still fails explicitly ("One of flow_eni_ids, flow_subnet_ids, flow_transit_gateway_ids, flow_transit_gateway_attachment_ids, or flow_vpc_ids must be provided.") when no target is supplied, instead of silently creating a partial deployment. The try(coalesce(...), []) fallback is kept only so tflint's all-defaults static evaluation doesn't crash; it no longer changes real runtime behavior. The no-target test now asserts that precondition failure instead of zero flow logs.

File: modules/aws/flow_logs/outputs.tf [hunk 3]
```diff
@@ -1,4 +1,24 @@
 output "arn" {
   description = "ARN of the cloudwatch log group used for flow logs"
   value       = aws_cloudwatch_log_group.log_group[*].id
-}
\ No newline at end of file
+}
+
+output "flow_log_ids" {
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/outputs.tf:6]: Fixed in 50a41bf9: added exact assertions on flow_log_ids and on each target-ID output, plus two new run blocks (with_transit_gateway_target_exposes_transit_gateway_output and with_transit_gateway_attachment_target_exposes_attachment_output) covering the previously-untested transit-gateway and attachment branches.

File: modules/aws/flow_logs/main.tf [hunk 4]
```diff
@@ -19,13 +19,18 @@ data "aws_region" "current" {}
...
     var.flow_eni_ids,
     var.flow_subnet_ids,
     var.flow_transit_gateway_ids,
     var.flow_transit_gateway_attachment_ids,
     var.flow_vpc_ids
-  )
+  ), [])
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/main.tf:38]: Fixed in 50a41bf9 — same fix as the earlier duplicate of this comment: aws_kms_key.key now carries a lifecycle precondition requiring a non-empty target list, so this stays an explicit plan-time failure rather than an empty flow_logs_source.

File: modules/aws/flow_logs/main.tf [hunk 5]
```diff
@@ -40,6 +47,14 @@ resource "aws_kms_key" "key" {
   key_usage                = var.key_usage
   is_enabled               = var.key_is_enabled
   tags                     = var.tags
+
+  lifecycle {
+    precondition {
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/main.tf:55]: Fixed in 78608446: bumped required_version to >= 1.2.0 (with an explanatory comment referencing lifecycle.precondition support), matching the pattern already used in modules/aws/ecs for a similar version-gated feature.

File: modules/aws/flow_logs/main.tf [hunk 6]
```diff
@@ -40,6 +47,14 @@ resource "aws_kms_key" "key" {
...
+      condition     = length(local.flow_logs_source) > 0
+      error_message = "One of flow_eni_ids, flow_subnet_ids, flow_transit_gateway_ids, flow_transit_gateway_attachment_ids, or flow_vpc_ids must be provided."
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/main.tf]: Fixed in 78608446: replaced the non-empty-only check with a count-based exactly-one-of-five precondition, plus a second precondition for the non-empty-list edge case (e.g. flow_vpc_ids = []). Good catch -- the multi-target case really would have hit a provider-level failure instead of a clear module error.

File: modules/aws/flow_logs/tests/flow_logs_source_resolution.tftest.hcl [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+    error_message = "flow_log_transit_gateway_ids should be null for a flow log targeted at a transit gateway attachment."
+  }
+}
+
+run "with_no_target_fails_explicit_precondition" {
+  command = plan
+
+  expect_failures = [
+    aws_kms_key.key,
+  ]
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/tests/flow_logs_source_resolution.tftest.hcl:184]: Fixed in 78608446: added the with_two_targets_fails_exactly_one_precondition run, which supplies both flow_vpc_ids and flow_subnet_ids and expects the (now count-based) precondition on aws_kms_key.key to fail.

File: modules/aws/flow_logs/outputs.tf [hunk 8]
```diff
@@ -1,4 +1,24 @@
...
+  description = "IDs of the created aws_flow_log resources. Useful for callers to verify wiring (that the flow log count and its target IDs match what was passed in)."
+  value       = aws_flow_log.this[*].id
+}
+
+output "flow_log_vpc_ids" {
+  description = "vpc_id of each created aws_flow_log resource, in the same order as flow_vpc_ids. Null entries indicate the flow log was targeted at a different resource type."
+  value       = aws_flow_log.this[*].vpc_id
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/outputs.tf:23]: Fixed in 78608446: added flow_log_eni_ids and flow_log_subnet_ids outputs (same null-fill pattern as the existing vpc/transit-gateway/attachment outputs), with test coverage for both target modes, and regenerated terraform-docs.

File: modules/aws/transit_gateway/connect/outputs.tf [hunk 9]
```diff
@@ -45,9 +45,13 @@ output "configurations" {
...
     bgp_asn                 = value.bgp_asn
     id                      = value.id
-    insider_cidr_blocks     = value.inside_cidr_blocks
+    inside_cidr_blocks      = value.inside_cidr_blocks
     peer_address            = value.peer_address
     transit_gateway_address = value.transit_gateway_address
+    # Deprecated: kept as a backward-compatible alias for the misspelled key
+    # this object previously exported. Use inside_cidr_blocks instead; this
+    # alias will be removed in a future major version.
+    insider_cidr_blocks = value.inside_cidr_blocks
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/transit_gateway/connect/outputs.tf:54]: Fixed in c4ea1ca1: moved the deprecation notice for insider_cidr_blocks into the public configurations output description and regenerated terraform-docs, so it's visible to callers, not just in an implementation comment.

File: modules/aws/flow_logs/main.tf [hunk 10]
```diff
@@ -19,13 +19,20 @@ data "aws_region" "current" {}
 ###########################
 locals {
 
-  flow_logs_source = coalesce(
+  # try() guards against the case where none of the five target variables are
+  # set (all null): coalesce() would otherwise fail with "no non-null,
+  # non-empty-string arguments" when static analysis tools (e.g. tflint)
+  # evaluate the module with all variables left at their defaults. Real
+  # callers must still provide exactly one target list -- that contract is
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/main.tf:29]: Fixed: updated the PR description in the current revision to explicitly call out the flow_logs functional behavior change (the exactly-one-target precondition contract) rather than describing it as output-only, and updated the Testing section to report the flow_logs test suite results.

File: modules/aws/flow_logs/variables.tf [hunk 11]
```diff
@@ -138,7 +138,7 @@ variable "iam_role_name_prefix" {
 variable "iam_role_permissions_boundary" {
   type        = string
   description = "(Optional) The ARN of the policy that is used to set the permissions boundary for the role."
-  default     = ""
+  default     = null
 }
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/variables.tf:142]: Fixed in 78608446: regenerated terraform-docs for flow_logs, which now correctly documents the default as null.

File: modules/aws/flow_logs/outputs.tf [hunk 12]
```diff
@@ -1,4 +1,24 @@
 output "arn" {
   description = "ARN of the cloudwatch log group used for flow logs"
   value       = aws_cloudwatch_log_group.log_group[*].id
-}
\ No newline at end of file
+}
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/outputs.tf:4]: Fixed in 78608446: changed the value to aws_cloudwatch_log_group.log_group[*].arn so the arn output actually returns the ARN, matching its description.

File: modules/aws/flow_logs/main.tf [hunk 13]
```diff
@@ -19,13 +19,20 @@ data "aws_region" "current" {}
...
+  # enforced explicitly by the lifecycle precondition on aws_kms_key.key
+  # below, rather than relying on coalesce() itself to error out.
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/main.tf:31]: Already fixed by the time this landed -- 78608446 (pushed just before this review ran) replaced the non-empty-only check with a count-based exactly-one-of-five precondition. Current main.tf enforces exactly one target, not just non-empty.

File: modules/aws/flow_logs/outputs.tf (hunk 8 above)
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/outputs.tf:23]: Already fixed by the time this landed -- 78608446 added flow_log_eni_ids and flow_log_subnet_ids outputs, completing coverage for all five target modes, with matching test coverage.

File: modules/aws/flow_logs/tests/flow_logs_source_resolution.tftest.hcl [hunk 14]
```diff
@@ -0,0 +1,119 @@
...
+    error_message = "Supplying flow_vpc_ids should still create exactly one flow log."
+  }
+
+  assert {
+    condition     = aws_flow_log.this[0].vpc_id == "vpc-0123456789abcdef0"
+    error_message = "The flow log's vpc_id should equal the supplied target."
+  }
+
+  assert {
+    condition     = length(output.flow_log_ids) == 1
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/tests/flow_logs_source_resolution.tftest.hcl]: Fixed in 1597c75a: replaced the length-only check with an exact comparison, output.flow_log_ids == aws_flow_log.this[*].id.

File: modules/aws/flow_logs/variables.tf [hunk 15]
```diff
@@ -138,7 +138,7 @@ variable "iam_role_name_prefix" {
 variable "iam_role_permissions_boundary" {
   type        = string
   description = "(Optional) The ARN of the policy that is used to set the permissions boundary for the role."
-  default     = ""
+  default     = null
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/variables.tf:141]: Already fixed by the time this landed -- 78608446 regenerated terraform-docs for flow_logs, which now correctly documents the default as null.

File: modules/aws/transit_gateway/connect/tests/connect.tftest.hcl [hunk 16]
```diff
@@ -169,14 +169,14 @@ run "multiple_connect_peers_expand_via_for_each" {
...
-  # NOTE: the configurations output's CIDR-block field is currently named
-  # insider_cidr_blocks (a typo -- it should be inside_cidr_blocks). That is
-  # fixed in the separate fix PR #409 (out of scope for this test-only PR
-  # since it requires editing outputs.tf), which also adds inside_cidr_blocks
-  # as the corrected key while keeping this one as a deprecated alias.
-  assert {
-    condition     = length(output.configurations["sdwan_vedge_1"].insider_cidr_blocks) == 1
-    error_message = "configurations output should expose exactly one CIDR block for this peer under its current (typo'd) key name."
+  assert {
+    condition     = length(output.configurations["sdwan_vedge_1"].inside_cidr_blocks) == 1
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/transit_gateway/connect/tests/connect.tftest.hcl]: Fixed in 1597c75a: replaced the length-only check with an exact-set comparison against the configured CIDR block(s) (toset() on both sides, since the underlying provider attribute is a set of string).

File: modules/aws/flow_logs/main.tf [hunk 17]
```diff
@@ -19,13 +19,20 @@ data "aws_region" "current" {}
...
+  flow_logs_source = try(coalesce(
     var.flow_eni_ids,
     var.flow_subnet_ids,
     var.flow_transit_gateway_ids,
     var.flow_transit_gateway_attachment_ids,
     var.flow_vpc_ids
-  )
+  ), [])
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/main.tf:38]: Confirmed accurate -- regional_nat_gateway_id was added to aws_flow_log in AWS provider v6.25.0 (this module's already-required minimum), so support could technically be added without a version bump. However, doing so is a genuine new feature (new variable, output, docs, and tests for a 6th target type), not a bug fix, so it doesn't fit this fix-scoped PR. Deferring to a separate feat(flow_logs) PR rather than expanding this one's scope further -- noted in the PR description. Leaving this open in case you'd like to track it as a follow-up issue.

File: modules/aws/flow_logs/main.tf [hunk 18]
```diff
@@ -1,5 +1,8 @@
 terraform {
-  required_version = ">= 1.0.0"
+  # >= 1.2.0: lifecycle.precondition (used below to enforce the "exactly one
+  # target" contract on aws_kms_key.key) was introduced in Terraform 1.2 /
+  # OpenTofu (all versions, since OpenTofu forked after Terraform 1.6).
+  required_version = ">= 1.2.0"
   required_providers {
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/flow_logs/main.tf:6]: Kept the required_version >= 1.2.0 bump and updated the PR to reflect it accurately: retitled to fix(vpc)!: ..., checked Breaking Changes = Yes, and added a BREAKING CHANGE footer explaining that Terraform 1.0/1.1 callers of modules/aws/flow_logs must upgrade (OpenTofu users are unaffected).

File: modules/aws/vpc/tests/vpc.tftest.hcl [hunk 19]
```diff
@@ -286,7 +286,10 @@ run "enable_nat_gateway_false_creates_no_nat_resources" {
 # that every NAT-gateway-dependent resource is skipped when no IGW/NAT can
 # exist. public_subnets_list is left at its non-empty default here so the
 # aws_route_table_association.public gating is meaningfully exercised (its
-# count must be 0 despite public subnets being present).
+# count must be 0 despite public subnets being present). aws_eip.nateip is
+# also asserted since this module additionally gates NAT EIP creation on
+# local.enable_igw (via local.enable_natgw) to avoid creating orphaned EIPs
+# when no NAT gateway can exist.
```
[the remark this answers is not in the document]
Comment (zachreborn) [reply modules/aws/vpc/tests/vpc.tftest.hcl:292]: Already fixed by the time this landed -- the PR description was rewritten to reflect the current diff (no more uniquely-named regression-test-file language; the actual comprehensive test files that were updated, vpc.tftest.hcl and connect.tftest.hcl, are what's described now).
